pages/contact/_input-hidden.html

```diff
@@ -0,0 +1,5 @@
+<div class="form-group">
+  <div class="row">
+      <input id="{{ include.id }}" name="{{ include.id }}" type="hidden" class="form-control">
+  </div>
+</div>
```

Comment:
If it's hidden, do we need the wrapper `div`s?

Reply:
Yeah, I thought about that but I didn't know if there was other Javascript relying on this somehow. I can try removing and see if that still works.